source/_integrations/squeezebox.markdown

```diff
@@ -31,11 +31,15 @@ The Squeezebox music player ecosystem, which can be controlled through this inte
 1. One or more [Squeezebox compatible hardware or software players](https://lyrion.org/players-and-controllers/).
 2. One or more [Lyrion Music Servers or Logitech Media Servers (LMS)](https://lyrion.org/getting-started) with the Squeezebox players connected to these servers.
 
-{% include integrations/config_flow.md %}
+## Supported devices
```

Comment:
are the supported devices part of the prereqs? or could they be placed above it? Like in the [doc structure example](https://developers.home-assistant.io/docs/documenting/integration-docs-examples#basic-structure)